modelkit/core/models/tensorflow_model.py

```diff
@@ -34,11 +34,45 @@
     from tensorflow_serving.apis.get_model_metadata_pb2 import GetModelMetadataRequest
     from tensorflow_serving.apis.predict_pb2 import PredictRequest
 
-except ModuleNotFoundError:
+except ModuleNotFoundError:  # pragma: no cover
     logger.info("Tensorflow serving is not installed")
 
 
-class TensorflowModel(Model[ItemType, ReturnType]):
+class AbstractTensorflowModel:
```

Comment:
NB: what about inheriting from `abc.ABC` ?

Reply:
In this instance, the methods are well defined. What would inheriting from ABC grant us?

Reply:
To be even more explicit on the fact that the user does not want to directly use an AbstractTensorflowModel, and be consistent with the class' name.

Reply:
I suppose that makes sense. Maybe we could call it an `AbstractTensorflowModelMixin` or something like that too